### ISSUE TITLE
Solar sensor level is not picked up by Boktai

### ISSUE THREAD

**Issue by dyamon** [libretro/mgba#216]
I'm trying to play [Boktai - The sun in your hand](https://en.wikipedia.org/wiki/Boktai%3A_The_Sun_Is_in_Your_Hand) and the solar sensor level (adjustable via the menu option) is not picked up by the game.

Note that the sensor emulation is picked up by the game in the initial configuration at the beginning of the game.

I noticed another closed issue #168 that was closed because of lack of interest. If it is a simple fix I might be able to do it myself but I have literally 0 experience with the codebase so I would need at least some pointers.

Version is: mGBA (0.8 5c03bbd1b7) coming with Batocera Linux 26

**Comment by endrift** [comment]
Did you make sure to set the solar sensor to the lowest level while doing the initial configuration? The game gets confused if you don't. You can recalibrate it in the settings menu of the game.

**Comment by dyamon** [comment]
Yes. During the configuration the option in the menu is set to 0. The configuration process is successful, but changes in the setting do not alter the light level in game.

**Comment by negativeExponent** [comment]
> Yes. During the configuration the option in the menu is set to 0. The configuration process is successful, but changes in the setting do not alter the light level in game.

you are already using a modified version of Boktai, those that were patched to use key combination to increase/decrease solar sensor levels. Those roms do not work with the emulated, solar sensor level offered in mgba.

**Comment by dyamon** [comment]
> you are already using a modified version of Boktai

then I should be able to change the solar level with some combination of keys. I tries L+Dpad, R+Dpad, R+A+Dpad, R+B+Dpad, Select+Dpad but nothing. I'll try with some other rom as well and report back.
